joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForDeclarationsCreator.scala

```diff
@@ -279,8 +279,16 @@ trait AstForDeclarationsCreator { this: AstCreator =>
     } else {
       MethodScope
     }
-    declaration.json("declarations").arr.foldLeft(Ast()) { (ast, d) =>
-      ast.merge(astForVariableDeclarator(d, scopeType))
+    val declAsts = declaration.json("declarations").arr.toList.map(astForVariableDeclarator(_, scopeType))
+    declAsts match {
+      case ::(head, tail) =>
```

Comment:
why not write this infix?
```suggestion
      case head :: tail =>
```